### ISSUE TITLE
Set Game Window Size does not center the window

### ISSUE THREAD

**Issue by Level0gamedev** [4ian/GDevelop#1371]
## Describe the bug
Using *Set Game Window Size* via action resizes the window correctly, however the window is resized from bottom right corner, making the window appear to the side and, depending on the chosen size, not all of the screen is usable

Please double check that the bug is not already reported in the issues list.

## To Reproduce
1. Set resolution in Project/Game Settings/Properties
2. Switch to event sheet
3. [At the beginning of the scene] -> [Set Game Window Size larger width and height (multiply by integer?)] 
4. Run game
5. Window will resize, but will no longer be centered

## Possible Fix
This might be expected behavior, but there should be an option to center the window (either through an option in the existing action, or a new action). 

Preferred solution is to add a secondary width and height in Properties for scaling automatically  (have an option for Game Size and Window Size - game size is the actual pixel size, window size allows to scale automatically. See Godot or Game Maker for similar behavior). This action would also eliminate the visible resizing when starting the game.

**Comment by 4ian** [comment]
I've decided for a separate action because it's the same to add the action just after the resizing compared to add a parameter - and it's more flexible in case other similar actions are added in the future.

I might add a separate property in the game settings for setting an initial window size that is different than the game resolution. For now, if you want to avoid the window changing size at startup:

* Set the game resolution size to the window size in game properties.
* Ensure you set up the scaling mode if you're doing pixel art :)
* at the beginning of the scene, set the game resolution back to a (probably smaller) size.

Closing but discussion can continue
